src/accelerate/utils/fsdp_utils.py

```diff
@@ -32,7 +32,32 @@
 logger = get_logger(__name__)
 
 
-def save_fsdp_model(fsdp_plugin, accelerator, model, output_dir, model_index=0):
+def _is_peft_model(model):
+    if is_peft_available():
+        from peft import PeftModel
+    unwrapped_model = getattr(model.module, "_orig_mod", model.module)
```

Comment:
do we have an unwrap model utility method in accelerate? If `model` is not a DDP `model.module` will fail here no?

Reply:
ok perfect then! maybe worht mentioning in a comment that `_is_peft_model` is only meant to be used for FSDP models, or maybe change the method name to `_is_fsdp_peft_model`

Reply:
I think `extract_model_from_parallel` in accelerate should cover this case if desired

Reply:
But worth double checking/writing a test :)

Reply:
Thanks @muellerzr, I've just updated this with the unwrapping utility and re-ran the test we have on the other branch on the multiple-GPU machine and it works.